t/chronicle.t

```diff
@@ -0,0 +1,63 @@
+use strict;
+use warnings;
+
+use DBI;
+use Test::MockTime qw(:all);
+use Test::More tests => 7;
+use Test::Exception;
+use Test::NoWarnings;
+use Data::Chronicle;
+use Date::Utility;
+use Test::Mock::Redis;
+
+my $redis = Test::Mock::Redis->new(server => 'whatever');
+my $dbh = DBI->connect( 'DBI:Mock:', '', '' )
+               || die "Cannot create handle: $DBI::errstr\n";
+
+my $d = { sample1 => [1, 2, 3],
+          sample2 => [4, 5, 6],
+          sample3 => [7, 8, 9] };
+
+my $d_old = { sample1 => [2, 3, 5],
+          sample2 => [6, 6, 14],
+          sample3 => [9, 12, 13] };
+
+my $first_save_epoch = time;
+
+my $chronicle = Data::Chronicle->new(
+    cache_writer    => $redis,
+    cache_reader    => $redis,
+    db_handle       => $dbh);
+
+is $chronicle->set("vol_surface", "frxUSDJPY", $d), 1, "data is stored without problem";
```

Comment:
please read the data back to see it's really stored or better check both the mocked database and the mocked redis.